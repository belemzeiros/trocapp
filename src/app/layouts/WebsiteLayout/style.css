html {
  font-family: Arial, Helvetica, "Liberation Sans", sans-serif;
}

label {
  display: block;
}
