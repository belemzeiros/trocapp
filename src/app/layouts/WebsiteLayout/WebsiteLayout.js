import React from 'react';
import PropTypes from 'prop-types';

import './style.css';

const WebsiteLayout = ({ route, ...props }) => (
  <React.Fragment>
    <route.page {...props} />
  </React.Fragment>
);

WebsiteLayout.propTypes = {
  route: PropTypes.element.isRequired,
};

export default WebsiteLayout;
