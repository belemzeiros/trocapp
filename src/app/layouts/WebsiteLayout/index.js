import WebsiteLayout from './WebsiteLayout';

export default WebsiteLayout;
