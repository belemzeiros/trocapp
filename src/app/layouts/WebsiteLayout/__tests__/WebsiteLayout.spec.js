import React from 'react';
import { createShallow } from '@material-ui/core/test-utils';
import WebsiteLayout from '../WebsiteLayout';

describe('WebsiteLayout', () => {
  const shallow = createShallow();

  it('WebsiteLayout will mount correctly', () => {
    const route = {
      page: jest.fn(),
    };
    const wrapper = shallow(<WebsiteLayout route={route} />);
    expect(wrapper).toMatchSnapshot();
  });
});
