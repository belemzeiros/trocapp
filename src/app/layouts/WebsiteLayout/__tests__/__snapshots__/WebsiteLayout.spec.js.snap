// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebsiteLayout WebsiteLayout will mount correctly 1`] = `
<React.Fragment>
  <mockConstructor />
</React.Fragment>
`;
